doc/display.xml

```diff
@@ -91,9 +91,9 @@
 
       This function was written by Attila Egri-Nagy and Manuel Delgado with some
       minor changes by J. D. Mitchell.<P/>
-      <Log><![CDATA[
+      <Example><![CDATA[
 gap> Splash(DotDigraph(RandomDigraph(4)));
-]]></Log>
+]]></Example>
```

Comment:
Actually I think this one needs to stay as `Log` instead of `Example`. This is intended to open a PDF viewer in the desktop environment, so it's not suitable for automated testing (and it's causing the `dot: not found` error in the CI).